### PR TITLE
Wu update includes

### PR DESCRIPTION
**Pull request by sophiaknowles**
`tsc` is the normal typechecking command we run for our projects. Unfortunately, it has a limitation that you can't typecheck specific files while using your custom `tsconfig.json` file. You either can typecheck a whole project using your `tsconfig.json` file, or typecheck specific files with no `tsconfig.json` specified. This npm package, [tsc-files](https://github.com/gustavopch/tsc-files), solves this problem for us by typechecking specified files with our specified `tsconfig`.

`tsc-files` creates a custom `tsconfig` (`const tmpTsconfig` here) that extends our current `tsconfig`. The update I'm making here is passing the files we want to typecheck in the `include` property of the extended `tsconfig` instead of in the `files` property. This is because in our current `tsconfig` we have the following:
```
  "include": ["src/**/*.ts", "src/**/*.js", "src/**/*.tsx", "src/**/*.jsx"],
  "exclude": ["node_modules", "*.json"],
  "files": ["react-table-config.d.ts", "custom.d.ts", "window.d.ts"]
```
If we pass in `files: [customfile1.tsx, customfile2.ts]`, then our ` "files": ["react-table-config.d.ts", "custom.d.ts", "window.d.ts"]` will be overridden which leads to typescript errors that we have setup our `*.d.ts` files to handle. I've made this update so that we pass in `include: [customfile1.tsx, customfile2.ts]` so that only these files are typechecked properly.

### CODE REVIEW

**Comment by idokatzwhoop** [comment]
Just double checking, we don't have to specify `"src/**/*.test.tsx"` right?

**Comment by idokatzwhoop** [comment]
Also, do you know if this would affect any other teams?

**Comment by sophiaknowles** [comment]
@idokatzwhoop 
> Just double checking, we don't have to specify "src/**/*.test.tsx" right?

Nope, we only want to be checking the explicit files that are passed in here 

> Also, do you know if this would affect any other teams?

It won't because I created this repo by forking the public version of it. No other teams are using this repo

**Review by idokatzwhoop — APPROVED** [review]
Makes sense to me... but would defer to a more senior dev to get their approval as well

**Comment by ericykim** [comment]
Does the readme need to be updated at all?

**Review by jkringel — APPROVED** [review]
Looks good. Can you also add to the `README` what you've changed so that other dev's are aware?

**Pull request merged**
